code/game/objects/structures/ladders.dm

```diff
@@ -19,12 +20,18 @@
 	return INITIALIZE_HINT_LATELOAD
 
 /obj/structure/ladder/Destroy()
+	if(up)
+		up.down = null

```

Comment:
You could technically have ladders that go down to different place.

Reply:
That'd be pretty terrible but good point